### PR TITLE
Fix a few remaining image expansions

### PR DESCRIPTION
**Pull request by sdodson**
Fixes https://bugzilla.redhat.com/show_bug.cgi?id=1570257

### CODE REVIEW

File: roles/openshift_node_group/defaults/main.yml [hunk 1]
```diff
@@ -41,7 +41,7 @@ l_openshift_prefix_dict:
   openshift-enterprise: 'ose-${component}'
 l_os_prefix: "{{ l_openshift_prefix_dict[openshift_deployment_type] }}"
```
Comment (michaelgugino) [inline roles/openshift_node_group/defaults/main.yml]: It appears l_os_prefix is not used any more?  Also l_openshift_prefix_dict ?

Same for other file and vars.

Comment (sdodson) [reply roles/openshift_node_group/defaults/main.yml]: Yea, removed them all.

**Comment by vrutkovs** [comment]
bot, retest this please

**Comment by sdodson** [comment]
Same flake

**Comment by sdodson** [comment]
https://github.com/openshift/origin/issues/19505

**Pull request merged**